LOAN/LoansSpecific/ConsumerLoans.rdf

```diff
@@ -64,32 +63,71 @@
+							<owl:Restriction>
+								<owl:onProperty rdf:resource="&fibo-fnd-pty-rl;isPlayedBy"/>
+								<owl:someValuesFrom rdf:resource="&fibo-be-le-lp;LegallyCompetentNaturalPerson"/>
+							</owl:Restriction>
+						</owl:unionOf>
+					</owl:Class>
 				</owl:someValuesFrom>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label xml:lang="en">consumer loan</rdfs:label>
-		<skos:definition xml:lang="en">loan whose borrower is person as opposed to an organization, i.e., a consumer</skos:definition>
+		<skos:definition xml:lang="en">loan to one or more individuals for household, family, or other personal expenditures</skos:definition>
+		<skos:example xml:lang="en">Other secured consumer loans may include loans against insurance policies, against securities held in a retirement account, loans secured with collateral other than for the loan&apos;s stated purpose, lines of credit other than credit card agreements, and so forth.</skos:example>
+		<cmns-av:adaptedFrom xml:lang="en">12 CFR § 228.12, https://www.law.cornell.edu/cfr/text/12/228.12</cmns-av:adaptedFrom>
+		<cmns-av:explanatoryNote xml:lang="en">In the United States, consumer loans do not include a home mortgage, small business, or small farm loans. They are defined by law to include the following: (1) Motor vehicle loan, which is a consumer loan extended for the purchase of and secured by a motor vehicle; (2) Credit card loan, which is a line of credit for household, family, or other personal expenditures that is accessed by a borrower&apos;s use of a &apos;credit card,&apos; as this term is defined in § 1026.2 of this chapter; (3) Other secured consumer loan, which is a secured consumer loan that is not included in one of the other categories of consumer loans; and (4) Other unsecured consumer loan, which is an unsecured consumer loan that is not included in one of the other categories of consumer loans.</cmns-av:explanatoryNote>
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-loan-spc-cns;HomeEquityLineOfCredit">
 		<rdfs:subClassOf rdf:resource="&fibo-fbc-dae-dbt;RevolvingLineOfCredit"/>
 		<rdfs:subClassOf rdf:resource="&fibo-loan-ln-ln;CollateralizedLoan"/>
-		<rdfs:subClassOf rdf:resource="&fibo-loan-spc-cns;ConsumerLoan"/>
+		<rdfs:subClassOf rdf:resource="&fibo-loan-spc-cns;SecuredConsumerLoan"/>
 		<rdfs:subClassOf>
 			<owl:Restriction>
 				<owl:onProperty rdf:resource="&fibo-fbc-dae-dbt;isCollateralizedBy"/>
 				<owl:someValuesFrom rdf:resource="&fibo-fbc-dae-dbt;PhysicalCollateral"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label xml:lang="en">home equity line of credit</rdfs:label>
-		<skos:definition xml:lang="en">line of credit secured by equity value in a borrower&apos;s home or other property</skos:definition>
+		<skos:definition xml:lang="en">line of credit granted to a homeowner secured by the equity value in a borrower&apos;s home or other property</skos:definition>
 		<cmns-av:abbreviation xml:lang="en">HELOC</cmns-av:abbreviation>
```

Comment:
HELOC -- more an acronym instead of an abbreviation.  Does rdf support acronyms?